types/serverless/index.d.ts

```diff
@@ -83,6 +83,15 @@ declare class Serverless {
     version: string;
 
     resources: AwsProvider.Resources;
+
+    configSchemaHandler: {
+        defineCustomProperties(props: any): any;
```

Comment:
you should define `addPropertiesToSchema` types since most function use this type.
Here is what I suggest :
- add a type called `JSONSchemaProperties` whose definition is 
```ts
properties: Record<string, unknown>;
required: string[]
```
- define `addPropertiesToSchema` prototype :
```
addPropertiesToSchema(schema: Record<string, unknown>, extension: JSONSchemaProperties): void
```

- use this type as well in your `defineCustomProperties` typings : 
```suggestion
        defineCustomProperties(schema: JSONSchemaProperties): void;
```